src/index.html

```diff
@@ -22,6 +22,377 @@
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__description">Product code: 195434</p>
+        <div class="mark">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="mark__reviews">Reviews: 5</p>
+        </div>
+        <div class="price">
+          <p class="price__word">Price:</p>
+          <p class="price__numeric">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="card-hover"
+          class="bottom"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="product-card__image"
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__description">Product code: 195434</p>
+        <div class="mark">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="mark__reviews">Reviews: 5</p>
+        </div>
+        <div class="price">
+          <p class="price__word">Price:</p>
+          <p class="price__numeric">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
+          class="bottom"
+        >
+          BUY
+        </a>
+      </div>
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="product-card__image"
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__description">Product code: 195434</p>
+        <div class="mark">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="mark__reviews">Reviews: 5</p>
+        </div>
+        <div class="price">
+          <p class="price__word">Price:</p>
+          <p class="price__numeric">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
+          class="bottom"
+        >
+          BUY
+        </a>
+      </div>
+      <div
```

Comment:
Add the blank line between the all cards
```suggestion
      </div>
      
      <div
```